brig/cmd/brig/commands/project_create.go

```diff
@@ -0,0 +1,537 @@
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/AlecAivazis/survey"
```

Comment:
👍

Reply:
@technosophos - what revision of this package did you use?
It's not in `Gopkg.toml`, and building off its latest revision currently fails